best-practices/howto_readme.md

```diff
@@ -9,18 +9,29 @@ All projects should have a `README.md` file. This file should try to answer the
 - How to deploy it (if appropriate)
 - How to run the tests
 - Include relevant badges:
-    - [Travis](http://docs.travis-ci.com/user/status-images/)
-    - [Gem Version](http://badge.fury.io/for/rb)
-    - Coverage ([codeclimate](https://codeclimate.com/))
-        - [Getting Started](https://docs.codeclimate.com/docs)
-
+    - Build status
+      - [TravisCI](http://docs.travis-ci.com/user/status-images/); or
+      - [CircleCI](https://circleci.com/docs/2.0/status-badges/)
+    - Current version (if a published package)
+      - E.g., if Ruby: [Gem Version](http://badge.fury.io/for/rb)
+    - Test coverage 
+      - [CodeClimate](https://codeclimate.com/); or
+      - Coveralls (though CodeClimate is preferred as of 2020)
+    - Code quality
+      - CodeClimate
+    - API validation (if project declares an HTTP API using the [OpenAPI Specification](http://spec.openapis.org/oas/v3.0.2))
+      - [OpenAPI validator](http://validator.swagger.io/validator)
+    - Docker image (if project publishes one or more docker images)
+      -  [Microbadger](https://microbadger.com/#badges)
```

Comment:
add gem version badge, if gem?

Reply:
@ndushay That's on line 16, but after I rejiggered things a bit which may make it easy to miss.